Documentation/WritingManPages.md

```diff
@@ -0,0 +1,89 @@
+# Guidelines for writing SerenityOS manual pages (manpages)
+
+## What does and does not belong in manual pages
+
+The SerenityOS manpages are the primary documentation for SerenityOS itself. Different from the documentation pages, it is not primarily intended for SerenityOS developers. If the question of "Would this information be useful to view within the OS?" can be answered with "Yes", that probably means the information belongs in a manual page.
+
+## Format
+
+Primary information on the manpage system and structure lives in [man(7)](../Base/usr/share/man/man7/man.md).
+
+Manpages are written in CommonMark Markdown. It is allowed, but not ideal, to use features not supported by Serenity's markdown tooling.
+
+### Sections
+
+Consult the primary structure information to choose the correct main section for a new page. If a section contains many closely related pages, or contains many pages in general, it may be worth moving them to a new subsection. Each subsection must have its own page which at minimum explains the purpose of the subsection and lists all pages contained within it.
+
+### Links
+
+It is allowed to link online resources in manpages.
+
+For resources specific to the manpage (for example, application screenshots), it is recommended to include these resources in the manpage folder next to the manpage. Other resources should be part of the SerenityOS [base files](../Base) and referenced via an absolute path (for example `/res/graphics/map.png`). Both kinds of paths will be converted to work correctly on [man.serenityos.org](https://man.serenityos.org).
+
+Links to other man pages must be provided via the `help` pseudo-scheme:
+
+```
+help://man/section/page
+```
+
+`section` is a main section number and `page` is a page name, possibly including sub-sections. For example: `help://man/1/Applications/FontEditor` or `help://man/7/boot_parameters`. Using this format is enforced by the manpage linter.
+
+### Headings
+
+The maximum heading level of manpages is 2 (`##`).
```

Comment:
This seems quite limiting. Can we break with convention and have more heading levels?

Reply:
You can have lower levels, just 1 is forbidden